pytorch_lightning/utilities/cli.py

```diff
@@ -34,6 +42,84 @@
     ArgumentParser = object
 
 
+class Registry(UserDict):
```

Comment:
Why subclass `UserDict` if its data field is unused?
```suggestion
class Registry(dict):
```

Reply:
Mark the class as protected? Users are not meant to manipulate it

```suggestion
class _Registry(dict):
```